public/index.html

```diff
@@ -1,11 +1,6 @@
 <html>
   <head>
     <title>Yelp Search Bar</title>
-    <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
```

Comment:
+1 for removing those and saving to node modules

Reply:
Yeah I think importing and exporting looks cleaner